lib/modules/subredditManager.js

```diff
@@ -176,6 +176,7 @@ module.go = async () => {
 	}
 
 	await getLatestShortcuts();
+	await getMultis();
```

Comment:
You could `const myMultis = await getMultis();` down below. You'd need to mark this function async and I think RES ajax handles de duplicating requests so you can get rid of the early return too.